### PR TITLE
feat: offsite backup sync via rclone sidecar

### LINKED ISSUE #10: Offsite backup sync to cloud storage (rclone sidecar)
## Context

- #4 introduced local automated dumps (implemented by #9): `pg_dump -Fc` files land in the dedicated `backups_data` volume.
- Local-only backups share fate with the host: server loss = data + backups gone together (no 3-2-1 redundancy).
- Goal: push dumps offsite automatically so a disaster recovery restore is possible even if the VPS dies.

## Proposed design

A second sidecar service `db-offsite` running `rclone/rclone`, decoupled from the dump loop:

```yaml
    db-offsite:
        image: rclone/rclone
        restart: unless-stopped
        depends_on:
            - db-backup
        volumes:
            - backups_data:/backups:ro          # read-only access to dumps
            - ./rclone.conf:/config/rclone/rclone.conf:ro
        entrypoint: [/bin/sh, -ec]
        command: |
            while :; do
                sleep "${OFFSITE_INTERVAL_SECONDS:-86400}"
                rclone copy /backups remote:pushup-backups --max-age 48h
                rclone delete remote:pushup-backups --min-age ${OFFSITE_RETENTION_DAYS:-30}d
            done
```

### Requirements

- [ ] New service in `compose.yml`, no change to the dump loop itself
- [ ] `/backups` mounted **read-only** in the sidecar
- [ ] Remote configured via `rclone.conf` bind mount (never commit it; add to `.gitignore`) — document `rclone config` creation steps in README
- [ ] Env knobs: `OFFSITE_INTERVAL_SECONDS` (default daily) and `OFFSITE_RETENTION_DAYS` (default 30, independent from local retention)
- [ ] Upload only recent dumps (`--max-age`) to avoid re-scanning everything each pass
- [ ] README section: which backend was chosen + how to restore from remote (`rclone copy remote:pushup-backups ./restore/` then standard `pg_restore` procedure)

### Backend candidates

| Backend | Cost | Notes |
|---|---|---|
| Backblaze B2 | ~free at this size | S3-compatible, simple |
| Scaleway Glacier | cheap | EU data residency |
| Any SFTP/NAS | existing hardware | no vendor account needed |

### Acceptance criteria

- [ ] A dump created locally appears on the remote within one sync interval
- [ ] Remote retention prunes independently of local retention
- [ ] Sidecar failure never impacts the dump loop or the bot
- [ ] Restore-from-cloud documented and tested once end-to-end

Related: #4 (local dumps), #9 (db-backup sidecar implementation).

### PR DESCRIPTION
**Pull request by LouisMoretti**
## Summary
- Add a `db-offsite` sidecar service (`rclone/rclone`) that pushes recent pg_dump dumps from the `backups_data` volume to `remote:pushup-backups` once a day, uploading only files newer than 48h (`--max-age`) and pruning remote copies older than `OFFSITE_RETENTION_DAYS` (default 30) with `--min-age`.
- `backups_data` is mounted read-only and the loop sleeps first, so the service never writes locally and no sync fires before dumps exist.
- `rclone.conf` (provider credentials, mounted read-only at `/config/rclone/rclone.conf:ro`) is added to `.gitignore` — never committed.
- README documents creating the remote with `rclone config`; `.env.example` documents `OFFSITE_INTERVAL_SECONDS` (default 86400) and `OFFSITE_RETENTION_DAYS` (default 30).
- No change to the existing db-backup dump loop.

Closes #10

### CODE REVIEW

**Comment by LouisMoretti** [comment]
## 🔍 Review — issue #10 (backup offsite rclone)

Diff vérifié intégralement :

**✅ Validé**
- Design conforme à l'issue : boucle `sleep ${OFFSITE_INTERVAL_SECONDS:-86400}` → `rclone copy --max-age 48h` → `rclone delete --min-age 30d`, `backups_data:/backups:ro`, bind `./rclone.conf:ro`
- Variables shell échappées en `$$` — cohérent avec db-backup existant, sinon compose aurait interpolé et cassé le conteneur
- Loop de dump existant : zéro touch
- `docker compose config -q` validé réellement, CI lint verte
- README (création remote, credentials, commandes de vérif) et .env.example complets ; rebase propre sur master

**🟡 Non bloquant** : si `./rclone.conf` n'existe pas au premier `docker compose up`, Docker le crée comme *répertoire* vide → le sidecar crashloop silencieusement. Une ligne de warning dans le README serait un plus (« create rclone.conf before first up »). Peut être fait dans un pass ultérieur.

Approuvé.

*Review postée en commentaire : GitHub interdit l'approbation formelle de sa propre PR.*
